docs/source/library-user-guide/functions/adding-udfs.md

```diff
@@ -345,12 +354,17 @@ async fn main() {
 }
 ```
 
-## Adding a Scalar Async UDF
+## Adding a Async Scalar UDF
 
-A Scalar Async UDF allows you to implement user-defined functions that support
+An `Async` Scalar UDF allows you to implement user-defined functions that support
 asynchronous execution, such as performing network or I/O operations within the
 UDF.
 
+:::{note}
+DataFusion `49.0.0`
+: This feature requires DataFusion version `49.0.0` or later.
```

Comment:
uh oh, no! This note might be useful today, but useless in the future.
Docs are always for the current version.
Release changelog is where version chaos is sorted out.

(I like when API docs document "since version" for every API element, but I assume that's always automated and applied consistently to all features, and sub-components, like new methods added later)